services/py-api/src/server/logger/logger_factory.py

```diff
@@ -88,13 +90,22 @@ def configure_app_logger(env: str) -> None:
                     CallsiteParameter.LINENO,
                 }
             ),
-            _CustomConsoleRenderer() if env == "DEV" else JSONRenderer(),
+            # If the ENV is DEV or PROD we save the logs in a JSON format. This is done in order to have better
+            # filtering and searchability
+            (JSONRenderer() if env in ("DEV", "PROD") else _CustomConsoleRenderer()),
         ],
         logger_factory=(
-            PrintLoggerFactory()
-            if env != "PROD"
-            else WriteLoggerFactory(file=Path("server").with_suffix(".log").open("a"))
+            # For DEV and PROD as these are VMs we save the logs to a logfile, so that we can check them later
+            # For LOCAL and TEST env we print the logs directly to the stdout
+            # FIXME: This could create an issue as the uvicorn logger will eventually create a server.log.1 file but
+            #  these logs will continue to be written to the old server.log file
```

Comment:
@asynchroza @Isak-Bego I'm not sure how to fix this from the top of my head. Another concern I have is the cluttering of the `server.log` on the DEV machine with health check logs. I have enabled DEBUG mode on line 109 for the DEV machine for easier debugging if there is a problem.